static/app/components/events/interfaces/spans/newTraceDetailsSpanDetails.tsx

```diff
@@ -599,6 +602,31 @@ function NewTraceDetailsSpanDetail(props: SpanDetailProps) {
   );
 }
 
+function SpanHTTPInfo({span}: {span: RawSpanType}) {
+  if (span.op === 'http.client' && span.description) {
+    const [method, url] = span.description.split(' ');
+
+    const parsedURL = safeURL(url);
+    const queryString = qs.parse(parsedURL?.search ?? '');
+
+    return parsedURL ? (
+      <Fragment>
+        <Row title={t('HTTP Method')}>{method}</Row>
+        <Row title={t('URL')}>
+          {parsedURL ? parsedURL?.origin + parsedURL?.pathname : 'failed to parse URL'}
+        </Row>
```

Comment:
Love it, but I think this should include the query parameters so people can copy/paste the full URL

Reply:
lets make a separate row - I think having it separate is nice bc you dont need to visually scan for the query string in the end

Reply:
Actually @gggritso the full URL is already displayed at the bottom!